.github/workflows/lint_and_test.yaml

```diff
@@ -1,11 +1,7 @@
 name: lint-and-test
 
 on:
-  pull_request_target:
-  push:
-    branches:
-      - main
-  workflow_dispatch:
+  pull_request:
   workflow_call:
```

Comment:
which one of these is actually required, when we want to run this during release?
https://github.com/open-component-model/ocm/blob/main/.github/workflows/release.yaml#L101

Reply:
`workflow_call` I believe.